featurebyte/api/feature.py

```diff
@@ -1123,36 +1124,30 @@ def validate_isin_operation(
         """
         assert_is_lookup_feature(self.node_types_lineage)
 
-    def isnull(self) -> Feature:
-        """
-        Returns a boolean Feature indicating whether each element is missing.
-
-        Returns
-        -------
-        Feature
-            Feature with boolean values
-
-        Examples
-        --------
-        >>> feature = catalog.get_feature("InvoiceCount_60days")
-        >>> new_feature = feature.isnull()
-        """
+    @substitute_docstring(
+        doc_template=ISNULL_DOC,
+        format_kwargs={"class_name": "Feature"},
+        examples=(
+            """
+            >>> feature = catalog.get_feature("InvoiceCount_60days")
+            >>> new_feature = feature.isnull()
+            """
+        ),
+    )
+    def isnull(self) -> Feature:  # pylint: disable=missing-function-docstring
         return super().isnull()
 
-    def notnull(self) -> Feature:
-        """
-        Returns a boolean Feature indicating whether each element is not null.
-
-        Returns
-        -------
-        Feature
-            Feature with boolean values
-
-        Examples
-        --------
-        >>> feature = catalog.get_feature("InvoiceCount_60days")
-        >>> new_feature = feature.notnull()
-        """
+    @substitute_docstring(
+        doc_template=NOTNULL_DOC,
+        format_kwargs={"class_name": "Feature"},
+        examples=(
+            """
+                >>> feature = catalog.get_feature("InvoiceCount_60days")
+                >>> new_feature = feature.notnull()
+                """
```

Comment:
nit: Would be good to align this properly in the coming PR.

Reply:
oh woops! yeap will fix in a follow-up!